config_scheduler.go

```diff
@@ -12,7 +12,7 @@ import (
 type SchedulerConfig struct {
 	TaskFinder                    string  `bson:"task_finder" json:"task_finder" yaml:"task_finder"`
 	HostAllocator                 string  `bson:"host_allocator" json:"host_allocator" yaml:"host_allocator"`
-	FreeHostFraction              float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
+	DefaultFutureHostPercent      float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
```

Comment:
Aren't all of these things defaults unless overridden in the distro? I'd argue for not putting it in the variable name.

Reply:
I think most of them are, but this until recently wasn't something that could be set on a distro level. However, I see your point.